src/commcare_cloud/manage_commcare_cloud/changelog.md.j2

```diff
@@ -0,0 +1,17 @@
+# About this changelog
+
+The following changes to `commcare-cloud` that require your attention,
+newest first.
+
+Changes that will require an action from anyone choosing
+to update on or after the date listed will be maked "_action required_".
+Those which are worth taking a look at but which may or may not require
+an action on your part will be marked "_action optional_".
+
+
+## Changelog
+{% for changelog_entry in changelog_contents %}
+### **{{ changelog_entry['date'] }}** [{{ changelog_entry['summary'] }}]({{ changelog_entry['filename'] }}) {% if changelog_entry['action_required'] %}(_action required_) {%endif %}
```

Comment:
This line (before endif) ends in a space, which means the line in the output ends in an unwanted space.
I would replace ` {%endif %}` here with a newline followed by `{%- endif %}`, which will trim away that newline.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/a516bc3342f8e0bded4bfa2bb1e07dba8169d99b